lib/basket/index.js

```diff
@@ -6,14 +6,30 @@
 var request = require('request');
 
 var config = require('../config');
+var logger = require('../logging')('basket');
 
 var API_KEY = config.get('basket.api_key');
 var API_URL = config.get('basket.api_url');
 var API_TIMEOUT = config.get('basket.api_timeout');
 
+// Proxy a request to the Basket backend and pipe onto the response
+exports.proxy = function basketProxy(path, options, res) {
+  return exports.request(path, options)
+    .on('error', function (error) {
+      logger.error(error);
+      try {
+        res
+          .status(500)
+          .json(exports.errorResponse(error, exports.errors.UNKNOWN_ERROR));
+      } catch (ex) {
+        logger.error(ex);
+      }
+    })
+    .pipe(res);
+};
```

Comment:
Although our future selves would probably appreciate a brief comment about why we're catching and ignoring exceptions here.